msmtpq-queue

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+QUEUE_DIR="${XDG_DATA_HOME:-$HOME/.local/share}/mail.queue"
+
+[[ -d "$QUEUE_DIR" ]] || mkdir -m0700 -p "$QUEUE_DIR"
+
+if [[ ! -w "$QUEUE_DIR" ]]; then
+    echo "Insufficient privileges to write to queue."
+    exit 1
+fi
+  
+if [[ ! -r "$QUEUE_DIR" ]]; then
+    echo "Insufficient privileges to read from queue."
+    exit 1
+fi
+
+HELP_TEXT="
+msmtpq-queue shows information about queued messages in
+the msmtpq-queue QUEUE_DIR.
+
+  current value: $QUEUE_DIR
+
+Usage:
+
+  -c    # print the count and exit
+  -p    # include the full message file path
+  -h    # this help
+
+"
+
+COUNT_ONLY=0
+SHOW_FILE_PATH=0
+# read in the options
+while getopts "cph" o; do
+    case "${o}" in
+        h)
+          # show help
+          echo "$HELP_TEXT"
+          exit 0
+          ;;
+        c)
+          COUNT_ONLY=1
+          ;;
+        p)
+          SHOW_FILE_PATH=1
+          ;;
+        *)
+          echo "$HELP_TEXT"
+          exit 1
+          ;;
+    esac
+done
+shift $((OPTIND-1))
+
+# This script shows the contents of the msmtpq queue directory
+
+QUEUE_COUNT=$(ls -A "$QUEUE_DIR" | wc -l)
```

Comment:
This should be `ls -1qA | wc -l` (print one file per line, quote special characters/whitespace/newlines).